JavaScriptCode.js

```diff
@@ -0,0 +1,368 @@
+var _ = require('lodash');
+const prompt = require('prompt-sync')();
+let str = true;
+let checkordernumber = () => {
+    while (str)
+    {
+        const ordernumber = prompt('Please enter your order : ');
+        if (!isNaN(ordernumber) && ordernumber.toString().indexOf('.') == -1 && ordernumber.toString().indexOf('#') == -1){
+            if(ordernumber >= 1){
+                return parseInt(ordernumber);
+                str = false;
+            }
+            else{
+                console.log(`The order number should be higher than 1.`);
+            }
+        }
+        else{
+            console.log(`Please enter a correct order number where order number should be an integer number higher than 1.`);
+        }
+    }
+}
+
+let myordernumber = checkordernumber();
+console.log(`Hey the order number is : ` + myordernumber);
+const largeboxobject = {size:'Large',value:20,price:1.80};
+const mediumboxobject = {size:'Medium',value:10,price:1.00};
+const smallboxobject = {size:'Small',value:5,price:0.6};
+const bagprice = 5.50;
+const currency = '$';
+let orderId = 1;
+
```

Comment:
let's split the functionality into multiple files instead of one file, with good naming

Reply:
noted

Reply:
done